qemu/tests/migration_with_file_transfer.py

```diff
@@ -38,6 +38,7 @@ def run(test, params, env):
     file_size = params.get("file_size", "500")
     transfer_timeout = int(params.get("transfer_timeout", "240"))
     migrate_between_vhost_novhost = params.get("migrate_between_vhost_novhost")
+    mig_file = "/home/migrate-%s" % utils_misc.generate_random_string(6)
```

Comment:
`test.tmpdir` should be better, + `os.path.join`.

Reply:
Yes, you're right, I have update.